journal-letters/original/03-epistles.adoc

```diff
@@ -1,8 +1,13 @@
 == Epistles Written by Thomas Wilson, And His Fellow-Labourer in the Service of the Gospel, James Dickinson, Whilst in America.
 
-=== An Epistle to Friends in Virginia and Carolina, from King`'s creek, in Maryland, the 12th of the Second month, 1692.
```

Comment:
This one will have to be given a short title too, like in the modernized.  And I just noticed that you made it a === (heading 3) instead of a heading 2.  I think this can be a heading 2 in both modernized and original.